src/CMakeLists.txt

```diff
@@ -4,13 +4,15 @@
 
 set(ROBOT_NAMES
     solo8
+    solo8ti
     solo12
     solo_simple_simu
     teststand
     stuggihop
 )
 
 set(ROBOT_CONFIG_FILES
+    "${robot_properties_solo_SOURCE_PREFIX}/config/dgm_parameters_solo8.yaml"
```

Comment:
Bug?

Reply:
Not sure if this this is a bug. In the `ROBOT_CONFIG_FILES`, you have to specify one yaml file for each robot. The robot order defined the lines above as `ROBOT_NAMES` is `solo8, solo8ti, ...`. So the `dgm_parameters_solo8.yaml` should show up twice.

Does this make sense?